testing/android-test-configurations/build.gradle.kts

```diff
@@ -24,4 +26,6 @@ pureKotlin()
 
 dependencies {
     testImplementation(libraries.protobufLite)
+    testImplementation(project(":testing:android"))
+    protoktExtensions(project(":extensions:protokt-extensions"))
```

Comment:
Are these additions needed? I think we may only need the extensions dependency.

Reply:
build fails without both dependencies